modules/mobile/app/controllers/mobile/v0/message_drafts_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require_dependency 'mobile/messaging_controller'
+
+module Mobile
+  module V0
+    class MessageDraftsController < MessagingController
+      def create
+        draft_response = client.post_create_message_draft(draft_params.to_h)
+        render json: draft_response,
+               serializer: Mobile::V0::MessagesSerializer,
+               status: :created
+      end
+
+      def update
+        create_draft_params = { message_draft: draft_params.to_h, id: params[:id] }
+        client.post_create_message_draft(create_draft_params)
+        head :no_content
+      end
+
+      def create_reply_draft
+        draft_response = client.post_create_message_draft_reply(params[:reply_id], reply_draft_params.to_h)
+        render json: draft_response,
+               serializer: Mobile::V0::MessagesSerializer,
+               status: :created
+      end
+
+      def update_reply_draft
+        create_reply_draft_params = { message_draft: reply_draft_params.to_h, id: params[:draft_id] }
+        client.post_create_message_draft_reply(
+          params[:reply_id], create_reply_draft_params
+        )
+        head :no_content
+      end
+
+      private
+
+      def draft_params
+        @draft_params ||= begin
+          params[:message_draft] = JSON.parse(params[:message_draft]) if params[:message_draft].is_a?(String)
+          params.require(:message_draft).permit(:category, :body, :recipient_id, :subject)
+        end
+      end
+
+      def reply_draft_params
+        @reply_draft_params ||= begin
+          # TODO: fix redundant JSON.parse maybe
```

Comment:
I assume you saw the comment in messages_controller about why this was needed - it came up specifically for messages with attachments that get sent as a multipart message. Are attachments relevant for drafts?

Reply:
Yes attachments can be added to drafts and draft replies according to the MHV documentation. I don't know any more regarding how it may be actually implemented, but that was the idea

Reply:
Correction I think I was thinking of replies. The attachment changes may be unnecessary